autogen/cluster_zonal.tf

```diff
@@ -204,7 +205,9 @@ resource "google_container_node_pool" "zonal_pools" {
     "initial_node_count",
     lookup(var.node_pools[count.index], "min_count", 1),
   )
-
+  {% if beta_cluster %}
+  max_pods_per_node  = lookup(var.node_pools[count.index], "max_pods_per_node", 110)
```

Comment:
Should this reference var. default_max_pods_per_node instead of harcoding 110?

Reply:
great point.

var.default_max_pods_per_node is a terraform attribute for google_container_cluster though. maybe we should be looking to only set max_pods_per_node if the node pool has max_pods_per_node set in the var.node_pools map?

Reply:
Agreed. We can likely use the Terraform 0.12 `nil` value for this use case.

Reply:
Great shout out, I didn't even know about that functionality! Added null as the default value for the lookup